.azure/pipelines/components-e2e-tests-new.yml

```diff
@@ -0,0 +1,45 @@
+# This configuration builds and runs Components E2E tests only
```

Comment:
Are we going to have both `components-e2e-tests.yml` and `components-e2e-tests-new.yml`?

Reply:
No, I only create the new one so I could invoke it manually (the old one is currently disabled, which means you can't even run it manually).

Before merging this PR, I'll replace the contents of the old pipeline with the new one, and delete the new one. Then we can re-enable the old one as per the "rehab plan" I listed above.

Thanks for checking.